PySDM/attributes/droplet/terminal_velocity/gunn_and_kinzer.py

```diff
@@ -67,7 +85,7 @@ def f4(r):
         def f8(r):
             return 1.104 * es + (1.058 * ec - 1.104 * es) * (6.21 + np.log(r)) / 5.01 + 1
 
-        @numba.njit(parallel=True)
+        @numba.njit()
         def terminal_velocity(values, radius, threshold):
             for i in prange(len(values)):
```

Comment:
parallel=True removed, but prange() remained - was it intentional?

Reply:
no, I think there should be `range()`, but `prange` works in the same manner if `parallel=False`